packages/boot/src/__tests__/acceptance/crud-rest-builder.acceptance.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {juggler, RepositoryMixin} from '@loopback/repository';
+import {RestApplication} from '@loopback/rest';
+import {CrudRestComponent} from '@loopback/rest-crud';
+import {expect, givenHttpServerConfig, TestSandbox} from '@loopback/testlab';
+import {resolve} from 'path';
+import {BootMixin, ModelApiBooter} from '../..';
+
+describe('CRUD rest builder acceptance tests', () => {
+  let app: BooterApp;
+  const SANDBOX_PATH = resolve(__dirname, '../../.sandbox');
+  const sandbox = new TestSandbox(SANDBOX_PATH);
+
+  beforeEach('reset sandbox', () => sandbox.reset());
+  beforeEach(givenAppWithDataSource);
+
+  afterEach(stopApp);
+
+  it('binds the controller and repository to the application', async () => {
+    await sandbox.copyFile(
+      resolve(__dirname, '../fixtures/product.model.js'),
+      'models/product.model.js',
+    );
+
+    await sandbox.writeTextFile(
+      'model-endpoints/product.rest-config.js',
+      `
+const {Product} = require('../models/product.model');
+module.exports = {
+  model: Product,
+  pattern: 'CrudRest',
+  dataSource: 'db',
+  basePath: '/products',
+};
+      `,
+    );
+
+    // Boot & start the application
+    await app.boot();
+    await app.start();
+
+    expect(app.getBinding('repositories.ProductRepository').key).to.eql(
+      'repositories.ProductRepository',
+    );
+
+    expect(app.getBinding('controllers.ProductController').key).to.eql(
+      'controllers.ProductController',
+    );
+  });
+
+  it('throws if there is no base path in the config', async () => {
+    await sandbox.copyFile(
+      resolve(__dirname, '../fixtures/product.model.js'),
+      'models/product.model.js',
+    );
+
+    await sandbox.writeTextFile(
+      'model-endpoints/product.rest-config.js',
+      `
+const {Product} = require('../models/product.model');
+module.exports = {
+  model: Product,
+  pattern: 'CrudRest',
+  dataSource: 'db',
+};
```

Comment:
Perhaps add a code comment to make it more clear that `basePath` was omitted here?

```js
 const {Product} = require('../models/product.model');
 module.exports = {
   model: Product,
   pattern: 'CrudRest',
   dataSource: 'db',
   // basePath not specified
 };
```